glean-core/python/glean/metrics/ping.py

```diff
@@ -17,6 +17,7 @@ def __init__(
         send_if_empty: bool,
         precise_timestamps: bool,
         include_info_sections: bool,
+        # enabled: bool,  -- Not currently enabled for python bindings
```

Comment:
Should we take any steps to ensure no one's trying to disable this on Python?

Reply:
I don't think it's too particularly risky as we don't have any Python consumers using Server Knobs or planning on using Server Knobs that I know of. The first use-case will likely be Cirrus and I'll be working closely with them on that so that's probalby when this will be enabled for Python.

Reply:
But can Python consumers disable a ping through `enabled: false` in the YAML?

Reply:
I should have stripped this from the output params, so it could right now but it would be easy enough to correct. I'll file a bug and fix that before we cut a glean parser release, but again, I don't see anyone using python making use of this property anytime soon.

Reply:
Or I can just enable it here, either way

Reply:
So, the _loader.py doesn't pass the `enabled` property to the constructor, so even if it found one in the pings.yaml file for a python Glean app, it wouldn't do anything with it. I think it's safe to leave as is for now, until time to enable this for python.